client/js/config/apollo/queires.js

```diff
@@ -0,0 +1 @@
+import {gql} from 'apollo-boost';
```

Comment:
Please write your queries in the container files. Thank you.

Reply:
removed quires.js